plugin/storage/grpc/config/config.go

```diff
@@ -37,72 +39,80 @@ type Configuration struct {
 	RemoteTLS            tlscfg.Options
 	RemoteConnectTimeout time.Duration `yaml:"connection-timeout" mapstructure:"connection-timeout"`
 	TenancyOpts          tenancy.Options
+}
+
+type ConfigV2 struct {
+	TenancyOpts                    tenancy.Options
```

Comment:
```suggestion
	Tenancy                    tenancy.Options
```
(why would we want "Opts" suffix in the config files?)